packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -84,7 +82,24 @@ if (
 const scope = Math.random().toFixed(16);
 const scopedSiteUrl = setURLScope(wordPressSiteUrl, scope).toString();
 const monitor = new EmscriptenDownloadMonitor();
-const wordPressModule = getWordPressModule(wpVersion);
+
+// Start downloading WordPress if needed
+let wordPressRequest = null;
+if (!wordPressAvailableInOPFS) {
+	if (requestedWPVersion.startsWith('http')) {
```

Comment:
If I'm reading this right, we now have support for custom WordPress builds?
Having the ability to prebuild WordPress with additional plugins and themes can be useful for demos like the Woo.com example we explored.

Reply:
You are reading this right indeed! :-)